lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -15,7 +15,7 @@ class DatabaseReplicationStandby < DatabaseReplication
 
     attr_accessor :disk, :standby_host, :run_repmgrd_configuration, :resync_data, :force_register
 
-    def initialize
+    def initialize(_hash = {})
```

Comment:
For this to work you need to actually initialize the parameters with the values in the hash.

Alternatively because they are already `attr_accessor`s you can just set them on the instance.